extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/sacn-bridge.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- sACN/E1.31 Bridge -->
+    <bridge-type id="sacn-bridge">
+        <label>sACN/E1.31 Bridge</label>
+        <description>The sACN/E1.31 bridge represents a single DMX universe connected via sACN/E1.31</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="mode" type="text">
+                <context></context>
+                <label>Transmission Mode</label>
+                <description>Set UDP mode to multicast (default)/unicast</description>
+                <required>true</required>
+                <options>
+                    <option value="unicast">Unicast</option>
+                    <option value="multicast">Multicast</option>
+                </options>
+                <default>multicast</default>
+            </parameter>
+            <parameter name="address" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of bridge in unicast mode, format: address[:port]. Default port is 5568.</description>
+                <required>false</required>
```

Comment:
Hard to tell what is correct here. In Multicast-Mode the receiver address is not needed, it is defined in the sACN standard. The address is needed in Unicat mode, though. Unfortunately it is not possible to define conditional "required" fields.